packages/nodes-base/nodes/Freshservice/descriptions/AgentDescription.ts

```diff
@@ -235,7 +235,8 @@ export const agentFields: INodeProperties[] = [
 				},
 			},
 			{
-				displayName: 'Member Names or IDs',
+				// eslint-disable-next-line n8n-nodes-base/node-param-display-name-wrong-for-dynamic-multi-options
+				displayName: 'Member of Group Names/IDs',
```

Comment:
In this file, I restored the original string since I find it unclear how to apply rule in this case.